roles/common/tasks/download.yml

```diff
@@ -1,4 +1,31 @@
 ---
+- name: Create calico release directory
+  local_action: file
+     path={{ local_release_dir }}/calico/bin
+     recurse=yes
+     state=directory
+  delegate_to: "{{ groups['kube-master'][0] }}"
```

Comment:
this has nothing related to kube-master
I suggest to use run_once: true and delegate to "downloader" host